NickvisionMoney.GNOME/Views/GroupDialog.cs

```diff
@@ -61,6 +60,9 @@ private GroupDialog(Gtk.Builder builder, GroupDialogController controller, Gtk.W
         _descriptionKeyController.SetPropagationPhase(Gtk.PropagationPhase.Capture);
         _descriptionKeyController.OnKeyPressed += (sender, e) => { if (e.Keyval == 59) { return true; } return false; };
         _descriptionRow.AddController(_descriptionKeyController);
+        //Apply button
```

Comment:
```suggestion
        //Apply Button
```

Sorry for nitpick, but that inconsistency was really bothering me :sweat_smile:

Reply:
I will later look over all comments to ensure consistency :)